calico-enterprise_versioned_docs/version-3.17/_includes/release-notes/_v3.17.0-release-notes.mdx

```diff
@@ -0,0 +1,7 @@
+xx May 2023
+
+### What's new {#whats-new-3.17.0}
+
+### Known issues {#known-issues-3.17.0}
+
+* Upgrading to {{prodname}} 3.17.0 on Rancher/RKE from {{prodname}} 3.13.0 currently requires manually terminating the calico-node container for upgrade to proceed.
```

Comment:
Are there additional release notes that need to be added ?

- What about MKE platform ? Users need to give rights to `cni-calico-plugin` service account for installation.
- We should also point out that we are introducing a new image and users should re-import the images into their private repositories.
- There are also some small updates to Elastic: indices will be created when we ingest data versus upfront; event index is currently linked to the previously created ILM event policy.

Reply:
> What about MKE platform ? Users need to give rights to cni-calico-plugin service account for installation.
Should this be in the Install instructions for MKE?

Reply:
> MKE might be worth adding here, since we give broad permissions. Could you propose a message?

> We should also point out that we are introducing a new image and users should re-import the images into their private repositories.
I don't think that is different from any other release.

> There are also some small updates to Elastic: indices will be created when we ingest data versus upfront; event index is currently linked to the previously created ILM event policy.
Not sure this can be categorized as a "Known issue" and also don't think that this implementation detail will be important enough to release note.

Reply:
This is what we say for MKE install. Is it enough to cover Alina's concern?
<img width="832" alt="image" src="https://github.com/tigera/docs/assets/48458529/05205c55-5c01-474a-b7f2-ae49b3f23af9">